styles.css

```diff
@@ -774,25 +944,564 @@ button:hover {
+    align-items: center;
+    margin: 0;
+    margin-top: -10%;
+    padding: 0;
+    height: 100%;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    margin: 10%;
+  }
+
+  .headline p {
+    font-size: 14px;
+    font-weight: 300px;
+    line-height: 20px;
+    width: 70%;
+    align-self: center;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    margin: 12%;
+    margin-top: 20%;
+  }
+
+  .containers {
+    width: 33.33%;
+  }
+
+  .container2 {
+    width: 33.33%;
+  }
+
+  .container3 {
+    width: 33.33%;
+  }
+
+  .contact1 {
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    visibility: visible;
+  }
+
+  .contact1 ul li {
+    width: 30%;
+    height: 10%;
+    border-radius: 0%;
+  }
+
+  .about-container img {
+    width: 60%;
+    background: none;
+  }
+
+  .contact1 ul li img {
+    width: 12px;
+  }
+
+  .next {
+    background-color: #fff;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  .h2 {
+    margin: 0;
+  }
+
+  .container1 {
+    width: 30.5%;
+    border: 1px solid #d0d9d4;
+    top: 7%;
+    margin: 5px;
+    padding: 0;
+    justify-content: center;
+    align-content: center;
+  }
+
+  .containera {
+    width: 30.5%;
+  }
+
+  .containerb {
+    width: 30.5%;
+  }
+
+  .containerc {
+    width: 30.5%;
+  }
+
+  .containerd {
+    width: 30.5%;
+  }
+
+  .containere {
+    width: 30.5%;
+  }
+
+  .text h2 {
+    width: 90%;
+    margin: 0;
+  }
+
+  .text ul {
+    width: 100%;
+    gap: 5px;
+    justify-content: center;
+  }
+
+  .text ul li {
+    width: fit-content;
+    padding: 5px;
+    padding-left: 0;
+    gap: 3px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    width: 100%;
+    overflow-y: hidden;
+  }
+
+  .about p {
+    margin-top: 10%;
+  }
+
+  .about button {
+    margin-top: 10%;
+  }
+
+  .left-corner {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-left: -180%;
+    margin-top: 1%;
+    width: max-content;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    gap: 5px;
+    margin: 0;
+    border-radius: 8px;
+    width: 100%;
+    padding-top: 0;
+    top: 0;
+    overflow-y: hidden;
+  }
+
+  #hidden {
+    display: none;
+  }
+
+  .skills ol {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: -50px;
+    margin-left: 15%;
+    gap: 3px;
+    margin-bottom: 20%;
+  }
+
+  .container4 {
+    width: 50%;
+    color: #172b4d;
+    align-self: self-start;
+    margin: 0;
+    padding-top: 5%;
+  }
+
+  .container5 {
+    width: 50%;
+    justify-content: left;
+    align-items: left;
+    margin-top: -20%;
+    margin-left: 40%;
+  }
+
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement as I mentioned by the previous code reviewer and also mentioned in my inline comments regarding your design not matching properly with the Figma design. Even your **contact-section** is collapsed completely and it seems like you need to understand the core concepts of CSS. Your design should start responsive from the `768px` screen size. Right now because as I can see you are not using the Correct `layout`, `media query`, and `spacing` around the About section and the font size of the About me section content is not correct. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-17 04-43-08](https://github.com/BHObserver/portfolio/assets/66476329/e5c0f09c-ac6d-4fd7-908d-5bb3ee1a207e)

</td>
<td>

![Screenshot from 2023-06-17 18-53-01](https://github.com/tsigewossne/portfolio/assets/66476329/827c0a61-9720-4e22-935b-572846deb667)

</td>
</tr>
</table>


**Hint**: *Please use **%age** for adjusting width with `margin: auto;` for making your container responsive and aligned centre always according to the viewport of the screen. Container which holds the child's content like headings, sub-headings, text or form. Don't use the fixed width or absolute units. Kindly try to understand the CSS positioning system along with box-model and you can find the lessons easily in the first week*.